iot-hub/Samples/service/JobsSample/JobsSample.cs

```diff
@@ -40,18 +41,26 @@ public async Task RunSampleAsync()
 
 
             // *************************************** Schedule twin job ***************************************
-            Console.WriteLine($"Schedule twin job {jobId} for {DeviceId}...");
-            JobResponse createJobResponse = await _jobClient
-                .ScheduleTwinUpdateAsync(
-                    jobId,
-                    query,
-                    twin,
-                    DateTime.UtcNow,
-                    (long)TimeSpan.FromMinutes(2).TotalSeconds);
-
-            Console.WriteLine("Schedule response");
-            Console.WriteLine(JsonSerializer.Serialize(createJobResponse, new JsonSerializerOptions { WriteIndented = true }));
-            Console.WriteLine();
+            try
+            {
+                Console.WriteLine($"Schedule twin job {jobId} for {DeviceId}...");
+                JobResponse createJobResponse = await _jobClient
+                    .ScheduleTwinUpdateAsync(
+                        jobId,
+                        query,
+                        twin,
+                        DateTime.UtcNow,
+                        (long)TimeSpan.FromMinutes(2).TotalSeconds);
+
+                Console.WriteLine("Schedule response");
+                Console.WriteLine(JsonSerializer.Serialize(createJobResponse, new JsonSerializerOptions { WriteIndented = true }));
+                Console.WriteLine();
+            }
+            catch (ThrottlingException)

```

Comment:
How about adding some doc reference in the catch block in case that the user would like to learn more about the quota?
https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-quotas-throttling#other-limits

Reply:
+1 to this. Whatever it took to help you understand this exception should be written here so that other users can learn about it as well